### DISCUSSION TITLE
Access to external storage Library

### DISCUSSION THREAD

**Discussion by aquaresima** [ampache/ampache#3475] [Q&A]
@girish 

Hi,
I am trying to add an external HD to my media folder, but not succeeding.

I installed Ampache with the docker (although I am not familiar with the docker infrastructure, nor with the issue that can arise with it). My docker is in `/home/cocconat/ampache/`
Once I installed the service all run smooth, and I can add the two test songs in the `
/home/cocconat/ampache/data/media/`

All amazing.

Now I would like to add to the Catalog my entire music library. It lives in an external storage mounted in `/home/cocconat/data/Music`
I tried to make a link to the folder with
```
sudo ln -s /home/cocconat/Music ampache/data/media/music
```
but the catalog does not see the linked folder,

I also tried to add a custom path in the   `Admin -> Add Catalog`, I chose `Local` catalog and use the abs path of my folder `/home/cocconat/data/Music` or the relative path `../../Music` which seems a bit silly ehehe. 

Anyway, no way to find this folder. 
My storage is an `exfat` mounted with fstab options: ```defaults,uid=1000,gid=1000,umask=022 0 1``` 
which should be fine if I do not want to upload/change the files.

Is this a problem? shall I add some user to my group? 

And how am I supposed to add the folder? via the link or somehow else?

Thanks, 
cocconat

**Comment by Khymick** [comment]
I do not use the docker, but I guess what you want to do can be solved by the external storage app in Nextcloud. You have to add your music dir as external storage.

**Reply by aquaresima** [reply]
Hi, sorry, what do you mean by nextcloud?

**Reply by Khymick** [reply]
Oh sorry, my wrong. I am setting up 2 different this at the moment: one time Ampache as stand-alone, the other thing is a music app under Nextcloud which is (as I believe somehow) also some kind of Ampache thing. I mixed up the two things, as I am posting things here as well in the Nextcloud forum 😀

**Reply by Khymick** [reply]
Nevertheless: as the Ampache Server is running under the Webserver user, does that user (such as “www-data”) have access to the music dir of yours?
